coupled_elasto_dynamics.cc

```diff
@@ -1276,7 +1278,17 @@ int main()
 
         const unsigned int dim = 3;
 
-        adapter::CoupledElastoDynamics<dim> elastic_solver("parameters.prm");
+        std::string parameter_file;
+        if (argc > 1)
+            parameter_file = argv[1];
+        else
+            parameter_file = "parameters.prm";
+
+        // Extract path for the output directory for the output directory
+        size_t pos = parameter_file.find_last_of("/");
```

Comment:
You could use  `boost::filesystem` to do this portably:
```
#include <boost/filesystem.hpp>
std::string case_path = boost::filesystem::path("/foo/bar.txt").parent_path();
```

Reply:
Filesystem is unfortunately not supported.